### DISCUSSION TITLE
PyScript any good?

### DISCUSSION THREAD

**Discussion by me21** [zauberzeug/nicegui#141] [Q&A] [answered]
I stumbled upon a project called PyScript: https://pyscript.net/

Would it add any value to NiceGUI to use it in the web pages?
I really don't know, as the project looks cool, but on another hand, NiceGUI hides the web page specifics from its user for the most part. Usually we don't need to write JavaScript, meaning we won't probably have to write PyScript either.

**Comment by falkoschindler** [comment, marked as answer]
PyScript is definitely an interesting project!
I'm just not sure what value it could bring to NiceGUI, whose core feature is already to do business logic in Python and synchronize the state to the client.
Nevertheless, I can imagine to create some `ui.pyscript` element some day, just because we can.
